src/components/common/Actions.vue

```diff
@@ -171,6 +176,15 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
       window.scrollTo({ top: 1250, behavior: 'smooth' })
       return
     }
+    // If it is a named company IA, validate NR before filing submission
+    if (this.isNamedBusiness && this.getNameRequestNumber) {
```

Comment:
You don't need both of these. See how they're declared.

Reply:
True. Will change it

Reply:
Done